CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,25 @@
+## General communication guidelines
+
+Ulauncher is a free, open source software project developed by a small team of volunteers and contributors.
+
+We are here because we want Ulauncher to improve, and for that we have to:
+* Act respectful towards each other.
+* Give each other the benefit of the doubt.
+* Give and accept constructive feedback.
+* Avoid arguments.
+* Learn from past mistakes.
+* Focus on what is best, not just for us as individuals, but for the Ulauncher community.
+* Refrain from trying to "win" arguments or advocate the importance of ones own specific needs.
+* Leave aside anything that is unhelpful for the Ulauncher progress, such as politics, non-constructive framing/rethorics or anecdotal arguments.
+
+We will not tolerate harassment, personal attacks, derogatory comments, trolling or uncivil behavior. Failure to comply will result in comments being hidden, moderated or removed, and on very serious or repeat offenses as well as refusal to comply users violating our terms will be blocked from the repository.
```

Comment:
I would break off another sentence here:

```
We will not tolerate harassment, personal attacks, derogatory comments, trolling, or uncivil behavior. Failure to comply will result in comments being hidden, moderated or removed. On very serious or repeat offenses users violating our terms will be blocked from the repository.
```